apps/metrics/templates/metrics/klass_metrics.html

```diff
@@ -282,7 +281,9 @@ <h5>Team Submission Times</h5>
                                 .done( (data) => {
                                     instructor_metrics.team_scores_chart.data.datasets[0].label = 'Team Scores'
                                     instructor_metrics.team_scores_chart.data.datasets[0].data = data.score
```

Comment:
Added some error handling to avoid weird chart behavior.

Reply:
👍